devices.js

```diff
@@ -9922,6 +9922,55 @@ const devices = [
             await bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
         },
     },
+    {
+        zigbeeModel: ['eTRV0100'],
+        model: 'eTRV0100',
+        vendor: 'Danfoss',
+        description: 'Danfoss Ally Thermostat',
+        supports: 'temperature, heating system control',
+        fromZigbee: [
+           fz.eurotronic_thermostat,
+           fz.battery,
+           fz.danfoss_thermostat_att_report,
+         ],
+        toZigbee: [
+            tz.thermostat_occupied_heating_setpoint,
+            tz.thermostat_local_temperature,
+            tz.danfoss_mounted_mode,
+            tz.danfoss_thermostat_orientation, tz.danfoss_algorithm_scale_factor,
+            tz.danfoss_heat_available, tz.danfoss_heat_request,
+            tz.danfoss_day_of_week,tz.danfoss_trigger_time,
+            tz.danfoss_window_open,
+            tz.danfoss_display_orientation,
+            tz.thermostat_keypad_lockout,
+        ],
+        meta: {configureKey: 3},
```

Comment:
Please add the required `exposes` property here (used for e.g. Home Assistant discovery). Search this file for examples from other TRVs.

Reply:
Will do, but it will have to wait for the weekend.